tools/bin/ecflow/processes/main/meteorological/get_metdata.py

```diff
@@ -0,0 +1,192 @@
+globe_attrs['note2'] = ("Citation: Abatzoglou, J.T., 2013, Development of gridded surface" + 
+		"meteorological data for ecological applications and modeling," +
+		"International Journal of Climatology, DOI: 10.1002/joc.3413") 
+globe_attrs['last_permanent_slice'] = "50"
+globe_attrs['note3'] = ("Data in slices after last_permanent_slice (1-based) are" + 
+		"considered provisional and subject to change with subsequent updates") 
+
+#latitude attributes
+lat_attrs = OrderedDict()
+lat_attrs['units'] = "degrees_north"
+lat_attrs['description'] = "latitude"
+
+#longitude attributes
+lon_attrs = OrderedDict()
+lon_attrs['units'] = "degrees_east"
+lon_attrs['description'] = "longitude"
+
+#time attributes
+day_attrs = OrderedDict()
+day_attrs['units'] = "days since 1900-01-01 00:00:00"
+day_attrs['calendar'] = "gregorian"
+day_attrs['description'] = "days since 1900-01-01"
+
+esri_str = ("GEOGCS[\\\"GCS_WGS_1984\\\",DATUM" +
+                "[\\\"D_WGS_1984\\\",SPHEROID\\\"WGS_1984\\\"," +
+                "6378137.0,298.257223563]],PRIMEM[\\\"Greenwich\\\",0.0]," +
+                "UNIT[\\\"Degree\\\",0.0174532925199433]]")
+
+#parameter attributes
+#precipitation
+pr_attrs = OrderedDict()
+pr_attrs['units'] = "mm"
+pr_attrs['description'] = "Daily Accumulation Precipitation"
+pr_attrs['_FillValue'] = -32767.
+pr_attrs['esri_pe_string'] = esri_str
+pr_attrs['coordinates'] = "lon lat" 
+pr_attrs['cell_methods'] = "time: sum(intervals: 24 hours)"
+pr_attrs['missing_value'] = -32767.
+
+#minimum temperature
+tmmn_attrs = OrderedDict()
+tmmn_attrs['units'] = "K"
+tmmn_attrs['description'] = "Daily Minimum Temperature"
+tmmn_attrs['_FillValue'] = -32767.
+tmmn_attrs['esri_pe_string'] = esri_str
+tmmn_attrs['coordinates'] = "lon lat" 
+tmmn_attrs['cell_methods'] = "time: sum(interval: 24 hours)"
+tmmn_attrs['height'] = "2 m"
+tmmn_attrs['missing_value'] = -32767.
+
+#maximum temperature
+tmmx_attrs = OrderedDict()
+tmmx_attrs['units'] = "K"
+tmmx_attrs['description'] = "Daily Maximum Temperature"
+tmmx_attrs['_FillValue'] = -32767.
+tmmx_attrs['esri_pe_string'] = esri_str
+tmmx_attrs['coordinates'] = "lon lat" 
+tmmx_attrs['cell_methods'] = "time: sum(interval: 24 hours)"
+tmmx_attrs['height'] = "2 m"
+tmmx_attrs['missing_value'] = -32767.
+
+#wind speed
+vs_attrs = OrderedDict()
+vs_attrs['units'] = "m/s"
+vs_attrs['description'] = "Daily Mean Wind Speed"
+vs_attrs['_FillValue'] = -32767.
+vs_attrs['esri_pe_string'] = esri_str 
+vs_attrs['coordinates'] = "lon lat" 
+vs_attrs['height'] = "10 m"
+vs_attrs['missing_value'] = -32767.
+
+
+#download metdata from http://thredds.northwestknowledge.net
+#precipitation
+pr_url = ("http://thredds.northwestknowledge.net:8080" +
+                "/thredds/dodsC/MET/pr/pr_%s.nc?lon[0:1:1385]," %(year) +
+                "lat[0:1:584],day[%s:1:%s]," %(num_day, num_day) +
+                "precipitation_amount[%s:1:%s]" %(num_day, num_day) +
+                "[0:1:1385][0:1:584]")
```

Comment:
I like how clear using the variable `num_day` makes the meaning of day and precipitation amount indices. Maybe set variables for 1385 and 584? (`num_lon` and `num_lat`?)